test-suite-consul-graal/build.gradle.kts

```diff
@@ -19,11 +19,12 @@ dependencies {
     testImplementation(mn.micronaut.json.core)
     testImplementation(mn.micronaut.http.client)
     testImplementation(mn.micronaut.http.server.netty)
-    testImplementation(mn.micronaut.jackson.databind)
+    testImplementation(mnSerde.micronaut.serde.jackson)
```

Comment:
Do we need to keep a separate module that tests with jackson?

Reply:
I extracted two modules one for Jackson Databind and one for serde